### ISSUE TITLE
Enable loading of vanilla loss functions for various frameworks

### ISSUE THREAD

**Issue by nrweir** [CosmiQ/solaris#31]
@nrweir commented on [Wed Apr 17 2019](https://github.com/CosmiQ/cw-nets/issues/48)

Required frameworks:
- [ ] Darknet (@avanetten is this necessary if we're going to use YOLT?)
- [ ] TensorFlow/Keras
- [ ] PyTorch

**Comment by nrweir** [comment]
Resolved with #57
